Makefile

```
@@ -750,8 +750,23 @@ endif
 ifdef ARTIFACT_DIR
 export JUNIT_OUTPUT_PATH = ${ARTIFACT_DIR}/junit_report.xml
 endif
+RESOLVE_COMPONENTS_SCRIPT := tests/e2e/scripts/resolve_affected_components.sh
+E2E_AUTO_RESOLVE ?=
 e2e-test:
```

checkmake (0.3.2)

[warning] Target body for "e2e-test" exceeds allowed length of 5 lines (14).

(maxbodylength)